javav2/example_code/iotsitewise/src/test/java/SitewiseTests.java

```diff
@@ -0,0 +1,261 @@
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestMethodOrder;
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.iotsitewise.model.CreateAssetModelResponse;
+import software.amazon.awssdk.services.iotsitewise.model.CreateAssetResponse;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueResponse;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class SitewiseTests {
+
+    private static final String assetModelName = "MyAssetModel"+ UUID.randomUUID();
+    private static final String assetName = "MyAsset";
+
+    private static String assetId = "";
+    private static final String portalName = "MyPortal";
+    private static String contactEmail = "";
+    private static final String gatewayName = "myGateway"+ UUID.randomUUID();
+    private static final String myThing = "myThing"+ UUID.randomUUID();
+
+    private static String assetModelId = "";
+
+    private static String iamRole = "";
+
+    private static final SitewiseActions sitewiseActions = new SitewiseActions();
+
+    private static Map<String, String> propertyIds;
+
+    private static String humPropId = "";
+
+    private static String tempPropId = "";
+
+    private static String portalId = "";
+
+    private static String gatewayId = "";
+    private static final String ROLES_STACK = "RoleSitewise";
+
+    @BeforeAll
+    public static void setUp() {
+        CloudFormationHelper.deployCloudFormationStack(ROLES_STACK);
+        Map<String, String> stackOutputs = CloudFormationHelper.getStackOutputsAsync(ROLES_STACK).join();
+        iamRole = stackOutputs.get("SitewiseRoleArn");
+
+         /*
+         The following values used in these integration tests are retrieved from AWS Secrets Manager.
+         */
+        Gson gson = new Gson();
+        String json = getSecretValues();
+        SecretValues values = gson.fromJson(json, SecretValues.class);
+        contactEmail = values.getContactEmail();
+    }
+
+    @Test
+    @Tag("IntegrationTest")
+    @Order(1)
+    public void testHelloService() {
+        assertDoesNotThrow(HelloSitewise::fetchAssetModels);
+        System.out.println(" Test 1 passed");
```

Comment:
Space between " and Test

Reply:
Or just remove all these "Test X passed" messages. The asserts should take care of that.

Reply:
fixed

Reply:
Did you miss a commit? I still see all the "Test X passed" messages